app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -515,4 +521,23 @@ def destroy
       ).deliver_later
     end
   end
+
+  def search
+    query = params.require(:query)
+    group_type = params.require(:groupType)
+    roles = roles_of_group_type(group_type)
+    roles = roles.select { |role| UserRole.is_active?(role) }
+
+    query.split.each do |part|
```

Comment:
What does this `split` do? Which char is it using to split the input query and why is that necessary?